meetings/Agenda_2017-09-03.md

```diff
@@ -0,0 +1,22 @@
+## Meeting Agenda, September 3, 2017
+
+1. **Fall events (5 min)**
+ * New meeting time: Thursday 6 pm (most popular slot)
+ * Possibly add Tuesday 4-6 as a coworking session? That way everyone who filled out the doodle can come. 
+ * Proposed events based on survey results:
+     - Intro Python
+     - Intro R
+     - Intro Git
+     - Advanced Git
+     - Gadgets and hardware: intro to programming small devices (Brian Sutherland, he requested November or December)
+     - Intro to machine learning concepts
+     - Intro to quantum computing concepts? Just for funsies
+     - Interactive plots and graphics for the web
+     - Intro to natural language processing
+     - scikit-learn - was this popular last time? It got a lot of hits on the survey
+
```

Comment:
Also Dr. Ju got back in touch about holding a workshop in either Last weeks of September (during weekends) or during fall break, it would be two 3-4 hr half days. They do all the booking and scheduling students as promised, we just provide the teaching. We can discuss whether this is too much on our plate or not with the RQM course...